pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/PulsarAdmin.java

```diff
@@ -154,6 +156,7 @@ public PulsarAdmin(URL serviceUrl, ClientConfiguration pulsarConfig) throws Puls
         WebTarget root = client.target(serviceUrl.toString());
         web = root.path("/admin");
 
+        this.internalConfiguration = new InternalConfigurationImpl(web, auth);
```

Comment:
should it be part of `brokers` as this configuration has been configured at broker (eg: `pulsar().getConfiguration().getxxx()`?

Reply:
- I was following how brokerStats is organized, which is not part of broker as well. 
- "brokers" is more about "brokers" in a cluster, not about internal stuffs for a broker. configuration here is more about "confiugration" in either cluster/broker level. I am feeling it is much better to have configuration stuffs in a separate endpoint.

let me know what is your opinons

Reply:
Ok..sure.. I am not strongly against of having separate resource.
Actually brokers already has `DynamicConfiguration` endpoint as well and internal-config also looks like config which is configured in broker rather a config/policies persisted in zk. So, felt to be part of brokers.